### DISCUSSION TITLE
Best practice to get notified, when f.e. nextcloud minor version changes

### DISCUSSION THREAD

**Discussion by JackHewson** [crazy-max/diun#861] [Q&A] [answered]
I'm watching f.e. nextcloud with diun like this:

```
- name: nextcloud:latest
```

Sometimes I get an notification, but the (minor) version of nextcloud doesn't have changed.
How to set up diun to get notified about minor updates (f.e. 25.0.2 -> 25.0.3)?

Thanks for your best practice instruction.

**Comment by crazy-max** [comment, marked as answer]
If you want to check [semver](https://semver.org/) releases you need enable `watch_repo` and also set a regular expression for tags to include. See the example for the file provider https://crazymax.dev/diun/providers/file/#example

Looking at the tagging strategy: https://hub.docker.com/_/nextcloud I think the following configuration would work for you:

```yaml
- name: nextcloud:latest
  watch_repo: true
  include_tags:
    - ^\d+\.\d+\.\d+$
```
